content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -282,6 +282,16 @@ on the Kubernetes API server for each static Pod.
 This means that the Pods running on a node are visible on the API server,
 but cannot be controlled from there.
 
+## Container probes
+
+A Probe is a diagnostic performed periodically by the kubelet on a Container. To perform a diagnostic, the kubelet calls a Handler implemented by the container. There are three types of handlers:
```

Comment:
Rule of thumb: if it's in the API but it's not listed by `kubectl api-resources`, either write it in backticks or in sentence case (not UpperCamelCase). Also, use italics when introducing new jargon.
So:
```suggestion
A _probe_ is a diagnostic performed periodically by the kubelet on a container. To perform a diagnostic, the kubelet can invoke different actions:
```

Reply:
Nice !! Learned new thing :)